docs/facts/known_issues.rst

```diff
@@ -121,3 +123,36 @@ to do this or manually run this SQL code insode your database.
 .. _SQLAlchemy: https://www.sqlalchemy.org/
 
 .. _dBeaver: https://dbeaver.io/
+
```

Comment:
Please create a PR with only this change. The rest should be handed inside the other PR:

Reply:
That already there, I just close this PR